llama_stack/apis/scoring_functions/scoring_functions.py

```diff
@@ -40,14 +73,13 @@ class ScoringFnDef(BaseModel):
         default_factory=dict,
         description="Any additional metadata for this definition",
     )
-    parameters: List[Parameter] = Field(
```

Comment:
Removing `parameters` field as we can just use `context` for defining parameters associated with the scoring function.